### PR TITLE
Update build docs

### PR DESCRIPTION
**Pull request by wangyum**

### CODE REVIEW

**Review by weiting-chen — APPROVED** [review]
Looks good to me.
Thanks for your contribution.

File: docs/developers/docker_gluten_centos7.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+```
+
+Run in docker:
+```bash
+git clone https://github.com/apache/incubator-gluten.git gluten
+cd gluten
+
+# To access HDFS or S3, you need to add the parameters `--enable_hdfs=ON` and `--enable_s3=ON`
+# If you have the same error with issue-3283, you need to add the parameter `--compile_arrow_java=ON`
+./dev/buildbundle-veloxbe.sh
```
Comment (wangyum) [inline docs/developers/docker_gluten_centos7.md:25]: `./dev/ci-velox-buildstatic-centos-7.sh` not work:
```
...
Could not find artifact org.apache.arrow:arrow-memory-unsafe:jar:15.0.0-gluten in central
...
```

`./dev/build_arrow.sh` also not work:
```
CMake Error at CMakeLists.txt:95 (project):
  No CMAKE_CXX_COMPILER could be found.

  Tell CMake where to find the compiler by setting either the environment
  variable "CXX" or the CMake cache entry CMAKE_CXX_COMPILER to the full path
  to the compiler, or to the compiler name if it is in the PATH.
```

**Comment by weiting-chen** [comment]
Which Gluten version are you using for docker testing?

**Comment by wangyum** [comment]
> Which Gluten version are you using for docker testing?

Master branch.

File: docs/developers/docker_gluten_centos7.md (hunk 1 above)
Comment (wangyum) [reply docs/developers/docker_gluten_centos7.md]: It's maybe my local env issue. `./dev/ci-velox-buildstatic-centos-7.sh` works if running on github action.

**Comment by weiting-chen** [comment]
Currently the website only put the document for official release such as v1.1.1(in archive section) or v1.2.0.
For main branch(v1.3), please visit the website(https://apache.github.io/incubator-gluten/) and submit a PR to modify the document in gluten source code repository(https://github.com/apache/incubator-gluten/tree/main/docs)
We have provided a docker image with docker script in main branch, please check below link: https://github.com/apache/incubator-gluten/blob/main/docs/developers/velox-backend-build-in-docker.md

File: docs/developers/docker_gluten_centos7.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+grand_parent: Documentations
+permalink: /docs/developers/docker-gluten-centos7/
+---
+Here is a docker script we verified to build Gluten+Velox backend on Pre-build CentOS 7:
+
+Run on host as root user:
+```bash
+docker pull apache/gluten:vcpkg-centos-7
+docker run -itd --name gluten-centos-7 apache/gluten:vcpkg-centos-7 /bin/bash
+docker attach gluten
```
Comment (wForget) [inline docs/developers/docker_gluten_centos7.md:15]:
```suggestion
docker attach gluten-centos-7
```

**Pull request closed without merging**